src/store/configureStore.dev.ts

```diff
@@ -1,28 +1,14 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import * as redux from 'redux';
-import {createOfflineReducer, networkStatusChangedAction, offlineCompose} from 'redux-offline';
-import defaultOfflineConfig from 'redux-offline/lib/defaults';
+import {createStore, applyMiddleware} from 'redux';
+import {composeWithDevTools} from 'redux-devtools-extension';
 import reducerRegistry from './reducer_registry';
-import devTools from 'remote-redux-devtools';
 
-const windowAny = window as any;
-
-const devToolsEnhancer = typeof windowAny !== 'undefined' && windowAny.__REDUX_DEVTOOLS_EXTENSION__ ? // eslint-disable-line no-underscore-dangle
```

Comment:
Note that I opted to depend on `redux-devtools-extension` to contain this logic.

Reply:
I'd react with thumbs up more times if I could. I've never been certain if anyone used this, and it's always bugged me how much console log spam it generates for something that might not be used.

Reply:
I definitely rely on Redux DevTools, and hope we can do so /more/ as we shift more logic to the Redux bus. So just to avoid ambiguity here -- the integration still exists -- but the `redux-devtools-extension` and it exposing `composeWithDevTools` just encapsulates all the magic here.